lib/ansible/modules/cloud/amazon/ec2_vpc_nat_gateway.py

```diff
@@ -66,6 +66,23 @@
       - Optional unique token to be used during create to ensure idempotency.
         When specifying this option, ensure you specify the eip_address parameter
         as well otherwise any subsequent runs will fail.
+  tags:
+    description:
+      - >
+        A dictionary of resource tags of the form: { tag1_name: 'tag1_value', tag2_name: 'tag2_value' }
+    required: false
+    aliases: [ "resource_tags" ]
+    version_added: "2.8"
```

Comment:
Does this have to be 2.9, now that 2.8 is entering release?

Reply:
2.10 :(